backend/routes/deepface_routes.py

```
@@ -0,0 +1,1320 @@
+                "title": alert_data.title,
+                "description": alert_data.description[:500],
+                "color": color,
+                "fields": [
+                    {"name": "Severity", "value": anomaly.get("severity", "medium").upper(), "inline": True},
+                    {"name": "Zone", "value": zone_id, "inline": True},
+                    {"name": "Alert ID", "value": str(alert.id), "inline": False},
+                ],
+                "timestamp": datetime.now(timezone.utc).isoformat(),
+            }],
+        }
+        asyncio.create_task(_post_discord_webhook(settings.DISCORD_WEBHOOK_URL, discord_payload))
```

Ruff (0.15.7)

[warning] Store a reference to the return value of `asyncio.create_task`

(RUF006)